pkgs/applications/graphics/komikku/default.nix

```diff
@@ -89,6 +89,7 @@ python3.pkgs.buildPythonApplication rec {
     description = "Manga reader for GNOME";
     homepage = "https://valos.gitlab.io/Komikku/";
     license = licenses.gpl3Plus;
+    platforms = platforms.linux;
```

Comment:
Usually we don't set meta.platforms for Python packages because they are not universal. The exception is when the package is explicitly not made for run on other platforms - say, a GUI for Linux `modprobe`.

In this case you are restricting this specific package to run on Linux only.
Is there any reason to it? The package looks like something that can run in any platform - it is just a manga reader, it does not rely on low level interfaces to OS...

Reply:
Hum... I didn't know about that for Python packages. I specified it following the [manual](https://nixos.org/manual/nixpkgs/stable/#var-meta-platforms) so it would have prebuilt binaries (not sure if I understood that right from the manual).

Taking a look at the [Guix builds](https://ci.guix.gnu.org/search?query=komikku%201.12.1%20spec:master&border-high-id=587618), they only build for Linux too (not that Guix runs on anything besides Linux and GNU Hurd), but well, I'm really not sure about it.

Should I revert my previous commit adding the `platforms` meta-attribute in that case?

Reply:
Indeed, `meta.platforms` is automatically set when you use `buildPythonPackage` (and most of specialized builders for langs like Ruby, Perl, Rust etc.). The advice you cited from the manual applies mostly to `stdenv.mkDerivation`.

Yes, please remove this specific commit from the stack (using git rebase or something like that).

Reply:
Done! @AndersonTorres, thank you for the review and for clarifying the `meta.platforms` attribute, I had no idea it was handled automatically in this case. :wink: